tempo_project/dataset/dimension_mappings.json5

```diff
@@ -0,0 +1,15 @@
+{
+  mappings: [
+    {
+      description: "ACS County 2018 to ACS County 2020 L48",
+      file: "dimension_mappings/county_to_county.csv",
+      dimension_type: "geography",
+    },
+    {
+      description: "TEMPO CONUS 2022 End Uses to TEMPO LDV Profiles 2022 End Uses",
+      dimension_type: "metric",
+      file: "dimension_mappings/metric_to_metric.csv",
+      mapping_type: "one_to_one",
+    },
+  ],
```

Comment:
I'm not sure how we're handling the non-publishing of 2018, 2020, 2022.
Looking at how counties were done, perhaps we should add a dimension_mappings/year_to_year.csv ?
Not sure of the dimension_type or mapping_type.

from_id,to_id
2018,
2020,
2022,
2024,2024
2026,2026

...

?